src/database/DataTypes/IndicatorAttribute.js

```diff
@@ -57,7 +57,7 @@ IndicatorAttribute.schema = {
     index: { type: 'int', default: 0 },
     isRequired: { type: 'bool', default: true },
     valueType: { type: 'string', default: 'string' },
-    valueDefault: { type: 'string', default: 'string' },
+    defaultValue: { type: 'string', default: 'string' },
```

Comment:
hmm, the default is the string `'string'`..?

Reply:
f3a91715917abf17f35a0f5ba07bf9b7d6cdde1c.